lib/chef/provisioning/docker_driver/driver.rb

```diff
@@ -105,7 +110,8 @@ def build_container(machine_spec, machine_options)
         target_repository = 'chef'
```

Comment:
I need test once again, but I think `docker_options[:from_image]` worked for me.